industrial_ci/src/docker.sh

```diff
@@ -162,7 +162,11 @@ function ici_build_default_docker_image() {
   ici_docker_build - <<EOF > /dev/null
 FROM ubuntu:$UBUNTU_OS_CODE_NAME
 
-RUN apt-get update -qq && apt-get -qq install --no-install-recommends -y wget ca-certificates sudo
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
```

Comment:
Setting DEBIAN_FRONTEND does not fix all the messages.

I have set a branch with the DEBIAN_FRONTEND to noninteractive [here](https://github.com/tecnalia-advancedmanufacturing-robotics/industrial_ci/blob/debconf_fix_test/industrial_ci/src/docker.sh#L165) , and then a [new test for kinetic](https://travis-ci.org/tecnalia-advancedmanufacturing-robotics/industrial_ci/jobs/223988245).

You can see in the [raw logs](https://s3.amazonaws.com/archive.travis-ci.org/jobs/223988245/log.txt) still some debconf messages complaining:

```
debconf: unable to initialize frontend: Dialog

debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76, <> line 125.)

debconf: falling back to frontend: Readline
```